operator-pipeline-images/operatorcert/static_tests/community/bundle.py

```diff
@@ -0,0 +1,90 @@
+import json
```

Comment:
Same here, please. It seems we are missing the pylint and mypy in the validation pipeline. I'll file a ticket for that but it will make the followup task easier if we add the docstring to a new code.